### PR TITLE
[ASM] - EXPANDER - 2805 - Azure Compute Tags

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[EXPANDR-2805](https://jira-dc.paloaltonetworks.com/browse/EXPANDR-2805)

## Description
Updating the Cortex ASM - Azure Enrichment playbook to retrieve tags from Azure Compute instances and saves to `asmtags` key.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg, please review

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_48.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex ASM - Azure Enrichment
+
+- Updated the playbook to retrieve tags from Azure Compute instances and saves to `asmtags` key.
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_48.md:6]: nit
```suggestion
- Updated the playbook to retrieve tags from Azure Compute instances and saves to `asmtags` field.
```

**Review by johnnywilkes — APPROVED** [review]
Approved, can change release notes or not

**Comment by rshunim** [comment]
Hi @BigEasyJ
Thanks a lot for your contribution,
Please move it from Draft to Ready for review when it's done

**Comment by ShirleyDenkberg** [comment]
@tomer-pan @johnnywilkes @rshunim Doc review completed.

**Comment by BigEasyJ** [comment]
@rshunim @tomer-pan please merge

**Review by rshunim — APPROVED** [review]

**Pull request merged**